src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -283,6 +391,51 @@ instruct vaddD(vReg dst, vReg src1, vReg src2) %{
   ins_pipe(pipe_slow);
 %}
 
+// vector add - predicated
+
+instruct vadd_masked(vReg dst_src1, vReg src2, vRegMask_V0 vmask) %{
+  match(Set dst_src1 (AddVB (Binary dst_src1 src2) vmask));
+  match(Set dst_src1 (AddVS (Binary dst_src1 src2) vmask));
+  match(Set dst_src1 (AddVI (Binary dst_src1 src2) vmask));
+  match(Set dst_src1 (AddVL (Binary dst_src1 src2) vmask));
+  ins_cost(VEC_COST);
+  format %{ "vadd.vv $dst_src1, $src2, $vmask\t#@vadd_masked" %}
+  ins_encode %{
+    BasicType bt = Matcher::vector_element_basic_type(this);
+    __ rvv_vsetvli(bt, Matcher::vector_length_in_bytes(this));
+    __ vadd_vv(as_VectorRegister($dst_src1$$reg),
+               as_VectorRegister($dst_src1$$reg),
+               as_VectorRegister($src2$$reg), Assembler::v0_t);
+  %}
+  ins_pipe(pipe_slow);
+%}
+
+instruct vaddF_masked(vReg dst_src1, vReg src2, vRegMask_V0 vmask) %{
```

Comment:
Can we combine this one with the next 'vaddD_masked' into a new instruct and give it a new name like 'vadd_fp_masked'? It looks to me that we can do similar thing for other instructs like 'vmulF_masked/vmulD_masked', etc.

Reply:
Fixed.